notebooks/DEKR_PoseEstimationFineTuning.ipynb

```diff
@@ -0,0 +1,1416 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Pose Estimation Fine Tuninig with Super Gradients\n",
+        "\n",
+        "In this tutorial notebook we demonstrate how to fine tune a pose estimation model using SuperGradients. It is recommended that you go over [Pose Estimation tutorial](https://docs.deci.ai/super-gradients/documentation/source/PoseEstimation.html) docs first to get familiar with terminology and concepts we use here.\n",
+        "\n",
+        "From this tutorial you will learn:\n",
+        "* How to implement a custom dataset class for pose estimation task\n",
+        "* How to instantiate a pre-trained pose estimation model and change number of joints it predicts to fit your dataset\n",
+        "* How to fine-tune a pose estimation model using SuperGradients\n",
+        "* How to visualize training progress and results in Tensorboard\n",
+        "\n"
+      ],
+      "metadata": {
+        "collapsed": false,
+        "id": "obAU90uggSAd"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install -q super_gradients==3.4.1 scikit-learn"
```

Comment:
Pls update to 3.5 🙏

Reply:
[50bf234](https://github.com/Deci-AI/super-gradients/pull/1593/commits/50bf23491b00609f522d253cbf37efbb71eeb1b7)